src/System.Globalization/tests/CompareInfo/CompareInfoTests.netstandard1.7.cs

```diff
@@ -297,17 +294,16 @@ public void SortKeyMiscTest()
             Assert.NotEqual(sk4.GetHashCode(), sk3.GetHashCode());
             Assert.NotEqual(sk4.KeyData, sk3.KeyData);
 
-            Assert.Equal(sk4, sk1);
-            Assert.Equal(sk4.GetHashCode(), sk1.GetHashCode());
-            Assert.Equal(sk4.KeyData, sk1.KeyData);
+            Assert.Equal(sk4, sk5);
+            Assert.Equal(sk4.GetHashCode(), sk5.GetHashCode());
+            Assert.Equal(sk4.KeyData, sk5.KeyData);
```

Comment:
Actually the test assumption was wrong but it happened to work on Windows. the test was written with the assumption of equality of sort key generated from lowercase string with case sensitive option and a sort key generated from same string but with upper casing and case insensitive option. this is wrong assumption but it happened to have Windows returned same key in such case. we cannot depend on this assumption on Windows as we can run into cases to break this assumption there. 

the correct thing is to compare sort keys generated from lower and upper casing with the case insensitive option.